core/Access.php

```diff
@@ -210,27 +200,42 @@ public static function getSqlAccessSite($select)
     }
 
     /**
-     * Reload Super User access
+     * Make sure a login name is set
      *
-     * @return bool
+     * @return true
      */
-    protected function reloadAccessSuperUser()
+    protected function makeSureLoginNameIsSet()
     {
-        $this->hasSuperUserAccess = true;
-
-        try {
```

Comment:
No longer needed to load all sites as we do now lazy load them. Performance win...